frappe/public/js/frappe/utils/pretty_date.js

```diff
@@ -77,7 +77,19 @@ function prettyDate(date, mini) {
 
 frappe.provide("frappe.datetime");
 window.comment_when = function (datetime, mini) {
```

Comment:
It will be horrible to use absolute date format everywhere. The requirement is only inside the timeline and form sidebar, so that the user can audit the changes easily without the need for hovering on the pretty date.

The `show_absolute_datetime_format` should be passed as an argument in this function. And we should pass it as true from the timeline and form sidebar.